init_conditions/get_subcell_agents.py

```diff
@@ -56,63 +57,117 @@ def download_max_cells_fov_images():
     '''
     download data for FOVs with most segmented cells and save XY projection as png
     '''
-    max_cells_fov_ids = get_max_cells_fov_ids()
-    pkg = quilt3.Package.browse("aics/hipsc_single_cell_image_dataset", "s3://allencell")
-    pkg_metadata = pkg["metadata.csv"]()
-    max_cells_fovs = pkg_metadata.loc[pkg_metadata["FOVId"].isin(max_cells_fov_ids)].sort_values(by=["FOVId"])
-    file_paths = np.unique(max_cells_fovs.loc[:,"fov_seg_path"].values)
-    with open("max_cells_fovs_metadata.csv", "w") as text_file:
-        text_file.write(max_cells_fovs.to_csv())
-    print("saved metadata for cells in FOVs with most segmented cells to max_cells_fovs_metadata.csv")
-    if not Path("aics_images").is_dir():
-        os.mkdir("aics_images")
-    for file_path in file_paths:
-        file_name = Path(file_path).name
-        if not Path(f"aics_images/{file_name}").is_file():
-            # download data
-            pkg["fov_seg_path"][file_name].fetch("aics_images/")
-        if not Path(f"aics_images/{file_name}.png").is_file():
-            # save XY projection of segmented FOV as png    
-            img = AICSImage(f"aics_images/{file_name}")
-            imageio.imwrite(f"aics_images/{file_name}.png", img.get_image_data("ZYX", S=0, T=0, C=1)[25])
+    if not Path("max_cells_fovs_metadata.csv").is_file():
+        max_cells_fov_ids = get_max_cells_fov_ids()
+        pkg = quilt3.Package.browse("aics/hipsc_single_cell_image_dataset", "s3://allencell")
+        pkg_metadata = pkg["metadata.csv"]()
+        max_cells_fovs = pkg_metadata.loc[pkg_metadata["FOVId"].isin(max_cells_fov_ids)].sort_values(by=["FOVId"])
+        file_paths = np.unique(max_cells_fovs.loc[:,"fov_seg_path"].values)
+        with open("max_cells_fovs_metadata.csv", "w") as text_file:
+            text_file.write(max_cells_fovs.to_csv())
+        print("saved metadata for cells in FOVs with most segmented cells to max_cells_fovs_metadata.csv")
+        if not Path("aics_images").is_dir():
+            os.mkdir("aics_images")
+        for file_path in file_paths:
+            file_name = Path(file_path).name
+            endname_ind = file_name.find(".")
+            file_name_noext = file_name[:endname_ind]
+            filedir = f"aics_images/{file_name_noext}"
+            if not Path(filedir).is_dir():
+                os.mkdir(filedir)
+                if not Path(f"{filedir}/{file_name}").is_file():
```

Comment:
nit: probably don't need this or line 83 now since you are always creating the filedir?